README-PLAYBOOK.md

```diff
@@ -95,7 +95,8 @@ The top-level structure of the file is described as follows:
 | credentials | [Credentials](#credentials) object | ***Required*** | ***Required*** | ***Required*** | A [Credential](#credentials) object that defines the credentials used to authenticate to the selected provider `platform`.                                                                                                                                                                |
 | platform    | string                             | ***Required*** | ***Required*** | ***Required*** | For TLS Protect Datacenter, either `tpp` or `tlspdc`.<br/>For TLS Protect Cloud, either `vaas` or `tlspc`.<br/>For Firefly, use `firefly`.                                                                                                                                                |
 | trustBundle | string                             | *Optional*     | n/a            | *Optional*     | Used when [Connection.platform](#connection) is `tlspdc` or `firefly`.<br/>Defines path to PEM-formatted trust bundle that contains the root (and optionally intermediate certificates) to use to trust the TLS connection. If omitted, will attempt to use operating system trusted CAs. |
-| url         | string                             | ***Required*** | *Optional*     | ***Required*** | URL of the Venafi platform to connect to.<br/>If url string does not include `https://`, it will be added automatically.<br/>For connection to TLS Protect Datacenter, `url` must include the full API path (for example `https://tpp.company.com/vedsdk/`.                               |
+| url         | string                             | ***Required*** | *Optional*     | ***Required*** | URL of the Venafi platform to connect to.<br/>If url string does not include `https://`, it will be added automatically.<br/>For connection to TLS Protect Datacenter, `url` must include the full API path (for example `https://tpp.company.com/vedsdk/` <br/> For TLS Protect Cloud use `https://api.venafi.cloud` (US region, default) or `https://api.api.venafi.eu` (EU region).                             
+|
```

Comment:
This should be in the same line, otherwise it will look printed after the table. Notice from your own fork:
![image](https://github.com/Venafi/vcert/assets/77068025/a30bc5bb-7d57-4358-b5b8-3c395980af57)